src/backend/tests/teams.test.ts

```diff
@@ -43,4 +45,36 @@ describe('Teams', () => {
     expect(prisma.team.findUnique).toHaveBeenCalledTimes(1);
     expect(prisma.team.findUnique).toHaveBeenCalledWith({ where: { teamId }, ...teamQueryArgs });
   });
+
+  describe('editTeam', () => {
+    test('setTeamMembers works', async () => {
+      jest.spyOn(prisma.team, 'findUnique').mockResolvedValue(prismaTeam1);
+      jest.spyOn(prisma.team, 'update').mockResolvedValue(prismaTeam1);
+      jest.spyOn(userUtils, 'getUsers').mockResolvedValue([superman, wonderwoman]);
+
+      const teamId = 'id1';
+      const userIds = [
+        {
+          userId: 2
+        },
+        {
+          userId: 3
+        }
+      ];
+      const res = await TeamsService.updateSingleTeam(flash, sharedTeam1.teamId, [2, 3]);
+
+      expect(prisma.team.findUnique).toHaveBeenCalledTimes(1);
+      expect(prisma.team.update).toHaveBeenCalledTimes(1);
+      expect(prisma.team.update).toHaveBeenCalledWith({
+        where: { teamId },
+        data: {
+          members: {
+            set: userIds
+          }
+        },
+        ...teamQueryArgs
+      });
+      expect(res).toStrictEqual(updatedSharedTeam1);
```

Comment:
beacuse you mocked prisma.team.update to return `prismaTeam1`, that's what res should be. it also means you don't need updatedSharedTeam1 at all which is good becuase we want to limit our test data

Reply:
I had to put sharedTeam1 to be equal to res since updateSingleTeam returns the team object after transforming it. In other words, it returns team obj with members and projects fields.